plugin/symbols.py

```diff
@@ -146,9 +154,6 @@ def on_symbol_selected(self, index: int) -> None:
         self.regions.clear()
 
     def on_highlighted(self, index: int) -> None:
-        if self.is_first_selection:
-            self.is_first_selection = False
-            return
```

Comment:
We had code that prevent scrolling to selected symbol on opening the quick panel but with this behavior change I think it's no longer needed. Otherwise the behavior would be inconsistent as the outline would not show initially but only after changing selection.